wavefront/server/modules/plugins_module/plugins_module/services/configuration_service.py

```
@@ -0,0 +1,172 @@
+        # namespace_service lives in agents_module, and importing it here would
+        # give plugins_module a cross-module dependency it does not have.
+        if not await self.namespace_repository.find_one(name=namespace):
+            raise NamespaceNotFoundError(
+                f"Namespace '{namespace}' does not exist. "
+                'Create it first via POST /v1/namespaces.'
+            )
+
+    def _invalidate(self, namespace: str, key: str) -> None:
+        try:
+            self.cache_manager.remove(get_configuration_cache_key(namespace, key))
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)